### PR TITLE
ALL: new data, invenio 3.1.1, issuses

### PR DESCRIPTION
**Pull request by rerowep**
* upgrads invenio to version 3.1.1
    * corrects the links on the README file.
    * closes rero-mef#33 (Variant name for person bad processed)
    * closes rero-mef#32 (gnd_pid / pid)
    * ignores VSCodium in .gitignore

### CODE REVIEW

File: Dockerfile.base [hunk 1]
```diff
@@ -51,6 +51,7 @@ RUN mkdir -p ${INVENIO_INSTANCE_PATH}
 
 # copy everything inside /src
 RUN mkdir -p ${WORKING_DIR}/src
+RUN mkdir -p ${WORKING_DIR}/sr
```
Comment (jma) [inline Dockerfile.base]: really?

File: rero_mef/api.py [hunk 2]
```diff
@@ -121,11 +120,11 @@ def create_or_update(
     def get_record_by_pid(cls, pid):
         """Get ils record by pid value."""
         assert cls.provider
-        resolver = Resolver(pid_type=cls.provider.pid_type,
-                            object_type=cls.object_type,
-                            getter=cls.get_record)
         try:
-            persistent_identifier, record = resolver.resolve(str(pid))
+            persistent_identifier = PersistentIdentifier.get(
```
Comment (jma) [inline rero_mef/api.py]: why? It seems less general than using a resolver. Is it for performance purpose?

File: rero_mef/api.py [hunk 3]
```diff
@@ -155,18 +154,37 @@ def get_persistent_identifier(cls, id):
     @classmethod
     def get_all_pids(cls):
         """Get all records pids."""
-        pids = [n.pid_value for n in PersistentIdentifier.query.filter_by(
-            pid_type=cls.provider.pid_type
-        )]
-        return pids
+        # pids = [n.pid_value for n in PersistentIdentifier.query.filter_by(
```
Comment (jma) [inline rero_mef/api.py]: you can remove the comments

File: rero_mef/api.py [hunk 4]
```diff
@@ -155,18 +154,37 @@ def get_persistent_identifier(cls, id):
...
+            yield persistent_identifier.pid_value
 
     @classmethod
     def get_all_ids(cls):
         """Get all records uuids."""
-        uuids = [n.object_uuid for n in PersistentIdentifier.query.filter_by(
-            pid_type=cls.provider.pid_type
-        )]
-        return uuids
+        # uuids = [n.object_uuid for n in PersistentIdentifier.query.filter_by(
```
Comment (jma) [inline rero_mef/api.py]: you can remove the comments

File: rero_mef/authorities/utils.py [hunk 5]
```diff
@@ -469,6 +388,7 @@ def write_link_json(
         agency_pid = viaf_pid
         add_schema(json_dump, 'viaf')
         json_dump['pid'] = agency_pid
+        del(json_dump['viaf_pid'])
```
Comment (jma) [inline rero_mef/authorities/utils.py]: why?  is viaf_pid has changed to pid?

File: scripts/setup [hunk 6]
```diff
@@ -63,47 +61,50 @@ function bulk_load () {
...
     do
-        pipenv run invenio records bulk_load ${AGENCY} ${INPUT_DIR} ${INPUT_DIR} --load_records --csv_pidstore_file ${AGENCY}_pidstore.csv --csv_metadata_file ${AGENCY}_metadata.csv --reindex --verbose
-        pipenv run invenio index run -c 4
+        pipenv run invenio fixtures bulk_load ${AGENCY} ${INPUT_DIR} ${INPUT_DIR} --load_records --csv_pidstore_file ${AGENCY}_pidstore.csv --csv_metadata_file ${AGENCY}_metadata.csv --verbose --reindex
     done
+    # Index agency records
     echo -e ${GREEN}"For faster indexing disable ES auto refresh:"${NC}
     echo "curl -X PUT \"localhost:29200/_settings\" -H 'Content-Type: application/json' -d '{\"index\" : {\"refresh_interval\" : \"-1\"}}'"
+    for AGENCY in rero bnf gnd mef
+    do
```
Comment (jma) [inline scripts/setup]: for loop seems useless as AGENCY is not used

File: Dockerfile.base (hunk 1 above)
Comment (rerowep) [reply Dockerfile.base]: ????

File: rero_mef/authorities/utils.py (hunk 5 above)
Comment (rerowep) [reply rero_mef/authorities/utils.py]: Because the `pid`of a record is called `pid`

File: rero_mef/api.py (hunk 2 above)
Comment (rerowep) [reply rero_mef/api.py]: To return the right record class.

**Pull request merged**